RedHatInsights/vulnerability-ui
# feat(RHIF-294): apply cve detail header filters  apply to both conventional and immutable systems

This PR enables the security rule filter and No vulnerable systems filter to be applied for both Conventional and immutable devices.

To test:

1. find a cve that has known exloits and also a security rule associated
2. Click on it and go to CVE detail page
3. Click on the filter for Security rule header card
4. observe that both tabs are filtered with it
5. Repeat the same thing with No vulnerable systems card
